debian/control

```diff
@@ -9,7 +9,7 @@ Build-Depends: gcc-9 | gcc-8 | gcc-7 (>=7.5),
  libcurl3-gnutls-dev | libcurl4-gnutls-dev | libcurl3-openssl-dev |
  libcurl4-openssl-dev | libcurl3-nns-dev | libcurl4-nns-dev, libgtest-dev,
  libflatbuffers-dev, flatbuffers-compiler, libglib2.0-dev, nnstreamer-tensorflow2-lite,
- nnstreamer-dev, libgstreamer1.0-dev, libgstreamer-plugins-base1.0-dev,
+ ml-inference-single-api-dev nnstreamer-dev, libgstreamer1.0-dev, libgstreamer-plugins-base1.0-dev,
```

Comment:
api.git/debian/control should be updated instead:

```
%package devel
Summary:	Tizen Native API Devel Kit for NNStreamer
Group:		Machine Learning/ML Framework
Requires:	capi-machine-learning-inference = %{version}-%{release}
Requires:	capi-machine-learning-inference-single-devel
Requires:	capi-machine-learning-inference-pipeline-devel
```

Reply:
api.git/debian/control is already:
```
Package: ml-inference-api-dev
Architecture: any
Multi-Arch: same
Depends: ml-inference-api, ml-inference-single-api-dev, ml-inference-pipeline-api-dev, ${shlibs:Depends}, ${misc:Depends}
```

Reply:
ml-api in debian is not getting update because of an ARM build error:

https://launchpadlibrarian.net/592924698/buildlog_ubuntu-bionic-armhf.ml-api_1.8.0.0-0~202203250540~ubuntu18.04.1_BUILDING.txt.gz

Reply:
https://github.com/nnstreamer/nnstreamer/issues/3698